lib/model/migrations/20221003-01-add-dataset-verbs.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2022 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+// backup.create and backup.terminate will be combined into a single verb,
+// config.set. If a role permitted backup.create but not backup.terminate (or
+// vice versa), that role will now permit both. However, that shouldn't be an
+// issue, because the two verbs should go hand-in-hand.
+
+const { without } = require('ramda');
+
+/* eslint-disable no-await-in-loop */
+
+const up = async (db) => {
+  // grant rights.
+  for (const system of [ 'admin', 'manager', 'viewer' ]) {
+    const [{ verbs }] = await db.select('verbs').from('roles').where({ system });
+    const newVerbs = verbs.concat([ 'dataset.list', 'dataset.read' ]);
```

Comment:
Should we add `entity.list`, `entity.create`, `entity.update` in here as well? Just like we have form.[verb] and submission.[verb], I think dataset's verbs and entities' verbs should be different

Reply:
Maybe we could replace the one place we use `dataset.read` with `entity.list`? I've suggested making that endpoint &hellip;/datasets/:name/entities.csv rather than &hellip;/datasets/:name/download to make the path more RESTful and to match the similar endpoint /projects/:projectId/forms/:xmlFormId/submissions.csv. Especially if we make that change, `entity.list` feels like a good verb to use.

If we introduced `entity.list`, I don't think we'd need `dataset.read` at all. As a parallel, there's a `field_key.list` verb but no `field_key.read`, since (at least right now) you can't read app users individually.

Above, I commented that I agree that it makes sense that there's no separate `dataset.update` verb, since that goes hand-in-hand with `form.update`. I think something similar is probably true of `entity.create` and `entity.update`. Entities are created via submissions, so I don't think there needs to be a verb other than `submission.create`. We don't check verbs in workers I believe, so it's also not clear to me where we'd check for a verb like `entity.create`.

Reply:
> Entities are created via submissions

At some point we may want to introduce those verbs to give certain roles the ability to create entities directly (e.g. bulk upload through Excel doc) but seems like we can add them then.

Reply:
I like the idea of adding and using `entity.list` instead of `dataset.read` now. It fits now and if we really need a different `dataset.read`, we could add that later, too, along with all the other eventual entity verbs. 

It'll address the RESTfulness of the csv download url in another PR!